services/movie.py

```
@@ -27,14 +32,20 @@ def create_movie(
     movie_description: str,
     genres_ids: list = None,
     actors_ids: list = None,
-) -> Movie:
-    movie = Movie.objects.create(
-        title=movie_title,
-        description=movie_description,
-    )
-    if genres_ids:
-        movie.genres.set(genres_ids)
-    if actors_ids:
-        movie.actors.set(actors_ids)
-
-    return movie
+) -> Movie | Exception:
```

The return type annotation `-> Movie | Exception` is misleading. According to the checklist, function signatures should accurately reflect their return behavior. Since this function raises exceptions instead of returning them, the annotation should be `-> Movie`.